### PR TITLE
texi2html 5.0

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by DomT4** [comment]
```

**Comment by DomT4** [comment]
```
==> /usr/local/Cellar/texi2html/5.0/bin/texi2html --help
Undefined subroutine &Locale::Messages::dgettext called at /usr/local/Cellar/texi2html/5.0/bin/texi2html line 29628.
Error: texi2html: failed
Failed executing: /usr/local/Cellar/texi2html/5.0/bin/texi2html --help
```

File: Library/Formula/texi2html.rb [hunk 1]
```diff
@@ -11,11 +11,12 @@ class Texi2html < Formula
     sha1 "2eefa2d9e010b7e6e9d67e2e847fa6d5ae9c9983" => :mountain_lion
   end
 
-  keg_only :provided_pre_mountain_lion
+  # keg_only :provided_pre_mountain_lion
```
Comment (DomT4) [inline Library/Formula/texi2html.rb]: Why the commenting out here?

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
It's building now, but the test is failing:
```
==> /usr/local/Cellar/texi2html/5.0/bin/texi2html --help
Undefined subroutine &Locale::Messages::dgettext called at /usr/local/Cellar/texi2html/5.0/bin/texi2html line 29628.
Error: texi2html: failed
```

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

File: Library/Formula/texi2html.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Texi2html < Formula
   desc "Convert TeXinfo files to HTML"
   homepage "http://www.nongnu.org/texi2html/"
-  url "http://download.savannah.gnu.org/releases/texi2html/texi2html-1.82.tar.gz"
-  sha256 "6c7c94a2d88ffe218a33e91118c2b039336cbe3f2f8b4e3a78e4fd1502072936"
+  url "http://download.savannah.gnu.org/releases/texi2html/texi2html-5.0.tar.gz"
+  sha256 "e60edd2a9b8399ca615c6e81e06fa61946ba2f2406c76cd63eb829c91d3a3d7d"
 
   bottle do
     cellar :any_skip_relocation
```
Comment (nijikon) [inline Library/Formula/texi2html.rb]: What is this for? Is it still needed?

File: Library/Formula/texi2html.rb [hunk 3]
```diff
@@ -14,9 +14,12 @@ class Texi2html < Formula
 
   keg_only :provided_pre_mountain_lion
```
Comment (nijikon) [inline Library/Formula/texi2html.rb]: What is this for? Is it still needed?

File: Library/Formula/texi2html.rb (hunk 2 above)
Comment (DomT4) [inline Library/Formula/texi2html.rb]: Yes. The CI will handle anything changed in the bottle block, including the notation on whether it needs relocation or not.

File: Library/Formula/texi2html.rb (hunk 3 above)
Comment (DomT4) [inline Library/Formula/texi2html.rb]: Unless 10.7 and below have suddenly removed the utility, yes :wink:

**Comment by nijikon** [comment]
Ok, so we can merge this /cc @DomT4

**Comment by DomT4** [comment]
Looks like it. Any idea why we haven't updated when this release has been around for 5 years?

**Comment by nijikon** [comment]
Don't have a clue :( If there will be complains we can always revert, and if something will get broken we can fix it.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by nijikon** [comment]
@DomT4 @bfontaine can we merge this?

**Pull request closed without merging**